src/test/java/org/openmrs/contrib/qaframework/RunTest.java

```diff
@@ -13,5 +14,6 @@
 		"html:target/index.html", "message:target/cucumber.ndjson"}, monochrome = true,
 // only scenarios tagged here are executed
 tags = "@selenium")
+
```

Comment:
it looks like we don't need any changes from this file

Reply:
Some things got messed up during conflict resolution. Am working out the best way and soon to address this

Reply:
you can clear the mess

Reply:
Have addressed them and committed the changes

Reply:
i can still see some formatting changes coming in